### ISSUE TITLE
Save Events

### ISSUE THREAD

**Issue by jamescooke** [jamescooke/prlint#22]
* Each PR call to the webhook is triggered on an event - save the Event linked to the Pull Request in the Database.

* Webhook view passes the created Event to the queue - the view doesn't care if the Event should be processed or not.

* Webhook view responds with the UUID of the Event, plus some other simplified rendering of the PR and Repo.

Example response:

```json
{
    "event": {
        "action": "opened",
        "pull_request": {
            "number": 27,
            "repository": {
                "full_name": "jamescooke/prlint",
                "name": "prlint",
                "remote_id": 53256881,
                "uuid": "0f7ab617-b628-4533-9daa-3fe8f3b90853",
            },
            "title": "Save events",
            "uuid": "4e86486c-c72b-49ec-b43e-c0d47762ebd8"
        },
        "state": "unprocessed",
        "uuid": "5a509b82-c6cc-4c57-8967-8f6f41c89f21"
    },
    "pr_created": true
}
```
